src/connection/connection_sr.c

```diff
@@ -1106,22 +1148,39 @@ css_connect_to_master_server (int master_port_id, const char *server_name, int n
     }
 
   /* select the connection protocol */
+
+  //  The following code is changed to register the server in the master_Server_monitor for monitoring
+  //  abnormally terminated servers (Check CBRD-24741 for details). This issue will be handled for
+  //  Linux/Unix first, and then for Windows. Therefore, the data sent to the master is different
+  //  between Windows and Linux/Unix. When Windows is supported, both of them will send proc_register
+  //  as register data.
```

Comment:
```suggestion
  //  TODO:
  // When supporting the Windows environment, It will be modified to send the same data (proc_register) for the Windows protocol (SERVER_REQUEST_NEW) as well.
```

Reply:
수정하였습니다.

Reply:
TODO 붙이는게 좋아보입니다. 추후 찾기 편합니다.